portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,18 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
```

Comment:
Here and elsewhere, prefer to use arrow functions. https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions